cluster/local/integration_tests.sh

```diff
@@ -85,7 +85,7 @@ check_pods(){
         sleep 10
         "${KUBECTL}" get events -A
         sleep 20
-        echo_error "unexpected number of pods ${count} ${1}"
```

Comment:
Why do we need this change?

Reply:
Yeah wasn't needed, removed already was just trying to figure out the e2e tests and accidentally added